packages/cli-lib/lib/watch.js

```diff
@@ -35,7 +41,9 @@ const _notifyOfThemePreview = (filePath, accountId) => {
 };
 const notifyOfThemePreview = debounce(_notifyOfThemePreview, 1000);
 
-function uploadFile(accountId, file, dest, options) {
+async function uploadFile(accountId, file, dest, options) {
+  const src = yargs.argv.src;
```

Comment:
I think there are still a couple of places where `yargs.argv` could be pulled from `options`